woocommerce/woocommerce-android
# [Custom Amounts M2] Redesign Payments and Customers section

Closes: #9941 


### Description

This PR introduces a restructured Payment section within the order creation workflow, aligning it with our design specifications. ichqhxDjbmZZmvJaJkGhkj-fi-118%3A6643

### Testing instructions


1. Navigate to Order creation screen
3. Check that the new design matches:

- Empty State (no items added)
- Add products, custom amounts, shipping, coupons, discounts, and gift cards to check that it works fine.
- Add customer details and notes to check that it works according to design as well.

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/1331230/5981d43c-cc88-4ebc-873c-a914118d2fc2
